### ISSUE TITLE
Proxy bypass option for Set-SNowAuth

### ISSUE THREAD

**Issue by ad-85** [insomniacc/PSSnow#58]
**Is your feature request related to a problem? Please describe.**
`Set-SNowAuth` appears to always use the default proxy settings when no URI + Credentials are passed in, however there is no switch for bypassing proxy entirely which our environment is set up to allow for ServiceNow.

**Describe the solution you'd like**
Ideally, a parameter added to `Set-SNowAuth` such as `-BypassProxy` which will allow for direct connections if desired with minimal-to-no impact to those relying on the existing behaviour.

**Additional context**

**Comment by insomniacc** [comment]
Thanks for the request, I'll get this added in 1.3.0 shortly

**Comment by insomniacc** [comment]
@ad-85 Sorry for the delay getting to this one, I know it's been a while but life has been crazy, only just managed to find some time to get back to my personal projects.
I've got a workable solution for PS Core users by adding a '-BypassDefaultProxy' flag to the Set-SNOWAuth command.
However theres really no easy way to do this for 5.1 users without introducing quite a lot of fundamental changes to the module which I'd rather not get into, so for that reason instead I'll put a warning message directing people to use either proxy bypass settings in windows or the 'no_proxy' environment variable which will achieve the same but is local machine config rather than something that this module can control. I'll document it all in the readme and put a link too.

I've got the code changes lined up and will release it later today.

Out of interest are you using this with 5.1 or Core?
